### PR TITLE
Added the emptyView back if not already in.

### PR DESCRIPTION
**Pull request by sunayna**
Make the emptyView visible if the list IS empty

### CODE REVIEW

File: src/java/com/sarality/app/view/list/EmptyListViewRenderer.java [hunk 1]
```diff
@@ -32,9 +32,14 @@ void render(ListView listView) {
 
   void render(LinearLayout layout, boolean isListEmpty) {
     View emptyView = getViewToRender();
+
+    if (layout.findViewById(emptyView.getId()) == null) {
```
Comment (abhideep) [inline src/java/com/sarality/app/view/list/EmptyListViewRenderer.java]: Why do we need to do this. The Empty view should always be there

**Comment by abhideep** [comment]
rest is fine

File: src/java/com/sarality/app/view/list/EmptyListViewRenderer.java (hunk 1 above)
Comment (sunayna) [inline src/java/com/sarality/app/view/list/EmptyListViewRenderer.java]: do we need the LinearLayout in here? Can I take it out ?

Comment (abhideep) [inline src/java/com/sarality/app/view/list/EmptyListViewRenderer.java]: Yes you do. How else would we show a NonScrollingListView. You just need to include an EmptyView next to it as well just like you would have done in a ListView.

Comment (sunayna) [inline src/java/com/sarality/app/view/list/EmptyListViewRenderer.java]: i mean the parameter "layout" in render method. it is not used.

**Pull request merged**